examples/dotnet-diagnostics/DiagnosticsExample/MainWindow.xaml.cs

```diff
@@ -157,4 +150,20 @@ await Dispatcher.InvokeAsync(() =>
             });
         }
     }
+
+    private async Task JoinToUserChannel()
+    {
+        if (await _desktopAgent.GetCurrentChannel() == null)
+        {
+            var channels = await _desktopAgent.GetUserChannels();
+            if (channels.Count() > 0)
```

Comment:
Nitpick: It's nicer to use Channels.Any() instead, or even just FirstOrDefault() and check if it's null